build/v1/types.go

```diff
@@ -746,6 +746,7 @@ type DockerBuildStrategy struct {
 	// volumes is a list of input volumes that can be mounted into the builds runtime environment.
 	// Only a subset of Kubernetes Volume sources are supported by builds.
 	// More info: https://kubernetes.io/docs/concepts/storage/volumes
+	// +listType=atomic
```

Comment:
these three markers contradict each-other

Reply:
updated in current push